win/logging.properties

```diff
@@ -0,0 +1,5 @@
+handlers=java.util.logging.FileHandler
+java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
+java.util.logging.FileHandler.pattern=%h/Library/Application\ Support/zcash4mac/zcash4mac.log
```

Comment:
This probably needs to be changed, we don't want to log to the zcash4mac log file 😈

Reply:
I'm not sure at this point if it's even used but I'll update it in case (to be later validated)